src/Project/Classes/Comparator/ProductComparator.java

```diff
@@ -0,0 +1,11 @@
+package Project.Classes.Comparator;

```

Comment:
[Fix] "classes" sunt toate in java si nu aduce nimic in plus acest pachet, se poate omite complet